sonar-project.properties

```diff
@@ -0,0 +1,13 @@
+#----- Project Configuration
+sonar.sources=src
+sonar.projectBaseDir=.
+sonar.projectVersion=1.0.0
+sonar.javascript.lcov.reportPaths=./coverage/lcov.info
+sonar.testExecutionReportPaths=__tests__/test-report.xml
+sonar.tests=__tests__
+sonar.test.inclusions=**/*.spec.ts
+sonar.coverage.exclusions=__tests__/**/*
+sonar.verbose=false
```

Comment:
Maybe for productive environments we might want to be verbose

Reply:
Yes, I thought about that, but since this is a base this should be fine for now.